exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,3 @@
 const name = " Daniel  ";
-
+const message = name.trim();
```

Comment:
Good use of the `.trim` method but the ask was to rewrite the same message as above.